spec/inferno/dsl/fhir_evaluation/rules/patient_85.json

```diff
@@ -0,0 +1,547 @@
+{
```

Comment:
Fixtures belong in `spec/fixtures`.

Reply:
Moved https://github.com/inferno-framework/inferno-core/pull/583/commits/1d03dee803bd9963f3da086c94b2ee4727d53bb0